aip/general/0009.md

```diff
@@ -108,12 +108,28 @@ A Network API exposed by a Google service. Most of these are
 hosted on the `googleapis.com` domain. It does not include other types of APIs,
 such as client libraries and SDKs.
 
+### IaC
+
+Short for Infrastructure-as-code, Iac describes a categories of clients that
+consumes a markup language or code that represents resources exposed by an API,
+and executes the appropriate imperative actions to drive the resource to that
+desired state.
+
+Examples of complexities that IaC clients abstract away include:
+
+- Ordering of creation of resources
```

Comment:
I would swap the order of these bullets and change this one to:
- Ordering of these imperative actions

Reply:
+1

Reply:
ack.